sql/database_cache.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
```

Comment:
The cache is nothing related to "session struct". It should be in a separate PR, one with thorough unit test and a benchmark to verify the performance improvement. Please remove the cache from this PR.

Reply:
Done, removed the `DBCache`.